packages/manager/apps/web/client/app/private-database/private-database.service.js

```diff
@@ -468,94 +468,23 @@ export default class PrivateDatabase {
   /*
         ORDER
     */
-  orderDuration(version, ram) {
```

Comment:
All those are deleted but I don't see any replacements in controller or whatever class that might call them. Were they not used ?

Reply:
Hello,

thanks for the review. I think it was used in legacy order private sql pages, but it's deprecated now.
see commit https://github.com/ovh/manager/pull/4865/commits/e0be9d37cf134b0e255d7cb8fd17b86bddc8df15